Bucketeer/Sources/SQLiteDatabase.swift

```diff
@@ -11,15 +11,15 @@ enum SQLiteError: Error {
 
 class SQLiteDatabase {
     private let dbPointer: OpaquePointer?
-    private let queue = DispatchQueue(label: "jp.co.cyberagent.Bucketeer.SQLite") // Serial
 
     private init(dbPointer: OpaquePointer) {
         self.dbPointer = dbPointer
     }
 
     static func open(path: String) throws -> SQLiteDatabase {
         var _db: OpaquePointer? = nil
-        if sqlite3_open(path, &_db) != SQLITE_OK {
+        let flags = SQLITE_OPEN_CREATE | SQLITE_OPEN_READWRITE | SQLITE_OPEN_FULLMUTEX
```

Comment:
There is no need to use a DispatchQueue to serialize the operations on the DB.
SQLite already supports serialized threading mode enabled by the flag `SQLITE_OPEN_FULLMUTEX`.